code/modules/mob/living/carbon/human/species/species_hud.dm

```diff
@@ -14,6 +14,7 @@
 	var/has_guns	=	TRUE	//Set to draw the gun/aiming toggle buttons
 	var/list/equip_slots = list() // Checked by mob_can_equip().
 	var/has_healthbar = FALSE	//Big healthbar across top of screen
+	var/has_resources = FALSE
```

Comment:
this var shouldn't exist, things might have more than one resource meter in future, they're designed to support an arbitrary quantity